src/clib/grackle_chemistry_data.h

```diff
@@ -136,6 +136,21 @@ typedef struct
   /* flag for Wolcott-Green+ 2011 H2 self-shielding */
   int H2_self_shielding;
 
+  /* flag to select which formula for calculating k11 you want to use. Setting to 1 will use Savin 2004, 0 will use Abel et al. 1997  */
+  int h2_charge_exchange_rate;
+
+  /* flag to select which formula for calculating h2dust you want to use. Setting to 1 will use Omukai 2000, 2 will use Hollenbach & McKee (1979) */ 
```

Comment:
Would it make sense to use 0 to turn this off?

Reply:
The h2dust coefficient is calculated each time (provided primordial_chemistry > 0). This parameter controls which formula you want to use (Omukai 2000 vs Hollenbach & McKee 1979). I think this should stay as 1 or 2, as either way h2dust is non-zero?